### PR TITLE
Update bufr to version 11.5.0 and ESMF to 8.1.1

### LINKED ISSUE #208: Need ESMF_8_1_1_beta_snapshot_01 for testing
**Is your feature request related to a problem? Please describe.**
Jun Wang and Moorthi Shrinivas reported performance issues with `ESMF_8_1_0_beta_snapshot_47` when used with threaded model execution. This might be an ESMF regression that went also out with the official 8.1.0 release. To test this hypothesis, we have prepared a new snapshot for a patch release `ESMF_8_1_1_beta_snapshot_01`, that needs to be tested.

**Describe the solution you'd like**
`ESMF_8_1_1_beta_snapshot_01` installed as part of hpc-stack on those platforms that Jun and Moorthi are doing their testing. Will need input from them.

**Additional context**
The snapshot is accessible at https://github.com/esmf-org/esmf/releases/tag/ESMF_8_1_1_beta_snapshot_01

### PR DESCRIPTION
**Pull request by kgerheiser**
Will fix #208 and #214 once installed

### CODE REVIEW

**Review by aerorahul — APPROVED** [review]
LGTM

File: config/stack_custom.yaml [hunk 1]
```diff
@@ -183,7 +183,7 @@ wrf_io:
 
 bufr:
   build: YES
-  version: bufr_v11.4.0
+  version: bufr_v11.5.0
   install_as: 11.4.0
```
Comment (aerorahul) [inline config/stack_custom.yaml]: @kgerheiser 
we missed this.
We want to also install as `11.5.0`

File: config/stack_gaea.yaml [hunk 2]
```diff
@@ -180,7 +180,7 @@ wrf_io:
 
 bufr:
   build: YES
-  version: bufr_v11.4.0
+  version: bufr_v11.5.0
   install_as: 11.4.0
```
Comment (aerorahul) [inline config/stack_gaea.yaml]: and here

File: config/stack_jedi.yaml [hunk 3]
```diff
@@ -181,7 +181,7 @@ wrf_io:
 
 bufr:
   build: YES
-  version: bufr_v11.4.0
+  version: bufr_v11.5.0
   install_as: 11.4.0
```
Comment (aerorahul) [inline config/stack_jedi.yaml]: here too

File: config/stack_mac.yaml [hunk 4]
```diff
@@ -184,7 +184,7 @@ wrf_io:
 
 bufr:
   build: YES
-  version: bufr_v11.4.0
+  version: bufr_v11.5.0
   install_as: 11.4.0
```
Comment (aerorahul) [inline config/stack_mac.yaml]: and everywhere

File: config/stack_noaa.yaml [hunk 5]
```diff
@@ -180,7 +180,7 @@ wrf_io:
 
 bufr:
   build: YES
-  version: bufr_v11.4.0
+  version: bufr_v11.5.0
   install_as: 11.4.0
```
Comment (aerorahul) [inline config/stack_noaa.yaml]: too many yamls

File: config/stack_ufs_weather_ci.yaml [hunk 6]
```diff
@@ -183,7 +183,7 @@ wrf_io:
 
 bufr:
   build: YES
-  version: bufr_v11.4.0
+  version: bufr_v11.5.0
   install_as: 11.4.0
```
Comment (aerorahul) [inline config/stack_ufs_weather_ci.yaml]: far too many.

**Review by aerorahul — CHANGES_REQUESTED** [review]
we missed the install_as update for bufr.

**Review by aerorahul — APPROVED** [review]
looks good.

**Comment by junwang-noaa** [comment]
@kgerheiser May I ask if the esmf 8.1.1 is now available on the supported platforms (dell, cray, hera, orion, jet)? Thanks

**Comment by kgerheiser** [comment]
Building it now. @Hang-Lei-NOAA and @climbfuji can you install ESMF 8.1.1?

**Comment by Hang-Lei-NOAA** [comment]
okay

**Comment by BinLiu-NOAA** [comment]
Thanks @kgerheiser and @Hang-Lei-NOAA! I saw esmf/8_1_1 is now available on Hera, Orion, wcoss_cray, and wcoss_dell_p3. I could not find it on Jet yet. Could one of you help to build it on Jet as well when you get a chance? Much appreciated!

**Comment by junwang-noaa** [comment]
@kgerheiser  May I ask if esmf811 is available on jet yet? We plan to update the model to use esmf811. Thanks.

**Comment by kgerheiser** [comment]
Yes, sorry. I missed the debug install.

**Comment by kgerheiser** [comment]
It's on Jet

**Comment by junwang-noaa** [comment]
Thanks, Kyle!

**Pull request merged**